localstack/services/awslambda/lambda_api.py

```diff
@@ -49,7 +49,7 @@
 LAMBDA_DEFAULT_HANDLER = 'handler.handler'
 LAMBDA_DEFAULT_RUNTIME = LAMBDA_RUNTIME_PYTHON27
 LAMBDA_DEFAULT_STARTING_POSITION = 'LATEST'
-LAMBDA_DEFAULT_TIMEOUT = 60
+LAMBDA_DEFAULT_TIMEOUT = 3000
```

Comment:
Looks like this value should be in seconds, rather than milliseconds?

Reply:
Confirm. The default timeout should be in seconds (checked via awscli, serverless and cloud formation).